mods/ts/rules/gdi-structures.yaml

```diff
@@ -142,12 +142,12 @@ GAWEAP:
 	RevealsShroud:
 		Range: 4c0
 	RallyPoint:
-		Offset: 4,1
+		Offset: 3,1
```

Comment:
You were giving misleading instructions at https://github.com/OpenRA/OpenRA/issues/10565#issuecomment-174167375 then, @pchote.